packages/amplify-category-custom/src/utils/dependency-management-utils.ts

```diff
@@ -29,7 +29,7 @@ export function getResourceCfnOutputAttributes(category: string, resourceName: s
   if (fs.existsSync(resourceBuildDir)) {
     const cfnFiles = glob.sync(cfnTemplateGlobPattern, {
       cwd: resourceBuildDir,
-      ignore: [AUTH_TRIGGER_TEMPLATE],
+      ignore: [AUTH_TRIGGER_TEMPLATE]
```

Comment:
Strangely, we are. Not sure why this formatted like this. As a general rule during these fixes, I am trying to avoid formatting files on save, and just fixing the lint rules. But prettier is in fact configured to keep those trailing commas.

FWIW, if you run the prettier formatter in the codebase currently, it fixes ~1500 files. So I'm just trying to avoid it altogether and let those get fixed organically as files are touched.

Reply:
The formatting is fixed

Reply:
looks like the trailing commas are still being removed. did a commit not get pushed?